### PR TITLE
Presence response

### PR DESCRIPTION
**Pull request by c0dzilla**
Fused responses to "who all in lab" into a single one.
Displays full names of members.

### CODE REVIEW

File: scripts/presence.coffee [hunk 1]
```diff
@@ -15,15 +15,17 @@
 
 module.exports = (robot) ->
   robot.respond /who.*lab/i, (msg) ->
+    message = ""
     msg.http("https://presence.sdslabs.co/?control=present").get() (err, res, body) ->
       if body is '[]'
-        msg.send 'No one here to give me company.'
+        meassage+='No one here to give me company.'
```
Comment (apsdehal) [inline scripts/presence.coffee]: Fix spacing issues everywhere

File: scripts/presence.coffee [hunk 2]
```diff
@@ -15,15 +15,17 @@
...
         if data.length is 1
-          msg.send 'I sense 1 human in lab.'
+          message+='I sense 1 human in lab.\n'
         else
-          msg.send 'I sense '+data.length+' humans in lab.'
+          message+='I sense '+data.length+' humans in lab.\n'
         for i of data
-          msg.send data[i].name.split(' ')[0]
-      msg.send "Here's a pic: https://presence.sdslabs.co/spycam.png?q="+Math.random()
+          message+=data[i].name+"\n"
```
Comment (apsdehal) [inline scripts/presence.coffee]: Have you tested it? 
Seems like it should work same as before.

Comment (c0dzilla) [inline scripts/presence.coffee]: @apsdehal earlier bot sent a different response for each person present in lab, alongside the image from presence. So there was a dirty result with some names above the image and some below. Now there is a single response containing everything.

File: scripts/presence.coffee [hunk 3]
```diff
@@ -15,15 +15,17 @@
...
-        msg.send 'No one here to give me company.'
+        meassage += 'No one here to give me company.'
       else
         data = JSON.parse(body)
         if data.length is 1
-          msg.send 'I sense 1 human in lab.'
+          message += 'I sense 1 human in lab.\n'
         else
-          msg.send 'I sense '+data.length+' humans in lab.'
+          message += 'I sense '+data.length+' humans in lab.\n'
```
Comment (apsdehal) [inline scripts/presence.coffee]: `' + data.length + ' human ` see the spacing

Comment (c0dzilla) [inline scripts/presence.coffee]: Corrected.

**Pull request merged**